docs/docs/working-with-video.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Working With Video*
+issue: https://github.com/gatsbyjs/gatsby/issues/12903
+---
+
+This is a stub. Help our community expand it.
+
+Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
+pull request gets accepted.
+
+The discussion around best practices for video in Gatsby is also ongoing on Github, chime in with your ideas here: https://github.com/gatsbyjs/gatsby/issues/3346
+
+---
+
+- [Sourcing video from a host](#sourcing-video-from-a-host)
+- [Embedding hosted videos in Markdown](#embedding-hosted-videos-in-markdown)
+- [Writing custom components for hosted video](#writing-custom-components-for-hosted-video)
+- [Querying video data from Markdown with GraphQL](#querying-video-data-from-markdown-with-graphql)
+- [Hosting your own HTML5 video files](#hosting-your-own-html5-video-files)
+- [Using custom video players](#using-custom-video-players)
+
+## Sourcing video from a host
+
+The easiest method for including video on a Gatsby site is to source an uploaded file from a site like YouTube, Vimeo, or Twitch. Using the source URL from one of those hosts, you can use Remark plugins or create a custom `<iframe>` solution to embed videos into your Gatsby site.
+
+## Embedding hosted videos in Markdown
+
+There are numerous Gatsby plugins for working with hosted video in your Markdown posts and pages. We recommend checking out the [gatsby-remark-embed-video](<(/packages/gatsby-remark-embed-video/?=video)>) plugin for sourcing from a variety of hosts.
```

Comment:
This link looks broken (see the `(<(`). I think some weird wrapping happened?

Reply:
Oh yeah, looks like something weird with my editor. Will fix!